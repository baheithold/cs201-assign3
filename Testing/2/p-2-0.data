1 1 130 ; 4 7 107 ; 5 1 462 ; 7 9 231 ; 9 3 423 ; 

