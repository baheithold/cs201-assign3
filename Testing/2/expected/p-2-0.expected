0: 1
1: 5(1)462
weight: 462
